pkg/bpf/bpf_kmesh_l4_workload.go

```diff
@@ -79,6 +84,16 @@ func (l *BpfLoader) StartWorkloadMode() error {
 	if err = l.workloadObj.Attach(); err != nil {
 		return fmt.Errorf("bpf Attach failed, %s", err)
 	}
+
+	if err = l.workloadObj.ApiEnvCfg(); err != nil {
+		return fmt.Errorf("failed to set api env")
+	}
+
+	ret := C.deserial_init()
+	if ret != 0 {
+		l.Stop()
```

Comment:
This is already done in the defer above

Reply:
there is a condition in defer action：err !=nil

Reply:
ok